packages/components/src/templates/next/components/internal/Wizgov/WizgovWidget.tsx

```diff
@@ -46,5 +46,13 @@ export const WizgovWidget = ({
     }
   }, [])
 
-  return <div id="wizgov-widget" data-agency={wizgovProps["data-agency"]} />
+  return (
+    <div
+      id="wizgov-widget"
+      data-agency={wizgovProps["data-agency"]}
+      {...(!!wizgovProps["data-topic"] && {
+        "data-topic": wizgovProps["data-topic"],
+      })}
```

Comment:
suggestion: we can spread the `wizgovProps` directly since they are the same. Any `undefined` will also not appear in the final output